bin/dev

```diff
@@ -8,4 +8,4 @@ fi
 # Default to port 3000 if not specified
 export PORT="${PORT:-3000}"
 
-exec foreman start -f Procfile.dev "$@"
+exec /C/Ruby32-x64/bin/ruby /c/Users/dell/AppData/Local/Microsoft/WindowsApps/foreman start -f Procfile.dev "$@"
```

Comment:
- [ optional] It would be fine if you can remove line 11 and replace it with  `exec foreman start -f Procfile.dev "$@"`, this will permit users to access your app with bin/dev command. 🙏

Reply:
@Oyepriscilla, kindly show me in the requirements where I was supposed to have implement that.

Reply:
@evansnyamekye kindly take note that I marked it as optional.